src/to-be-visible.js

```diff
@@ -1,33 +1,63 @@
 import {checkHtmlElement} from './utils'
 
-function isStyleVisible(element) {
+function getElementVisibilityStyle(element) {
+  if (!element) return 'visible'
   const {getComputedStyle} = element.ownerDocument.defaultView
+  const {visibility} = getComputedStyle(element)
+  return visibility || getElementVisibilityStyle(element.parentElement)
```

Comment:
Why do you walk up the tree here? `visibility` should never be falsy unless the user agent has an incorrect implementation of the CSS spec. [`visibility` has an initial value of `visible`](https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#formal_definition)